src/inline-svg.directive.ts

```diff
@@ -141,7 +141,7 @@ export class InlineSVGDirective implements OnInit, OnChanges, OnDestroy {
   private _insertEl(el: Element) {
     if (this.replaceContents && !this.prepend) {
       if (this._prevSVG) {
-        this._prevSVG.remove();
+        this._el.nativeElement.removeChild(this._prevSVG);
```

Comment:
Anyway, I have to call it a day today, so I will update the PR tomorrow. But if you prefer to do it in the meantime, feel free to close and just push a fix directly to master :)